fn/collation.xml

```diff
@@ -0,0 +1,955 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<test-set xmlns="http://www.w3.org/2010/09/qt-fots-catalog"
+          name="misc-UCACollation" covers-40="PR1093">
+  <description>Tests for fn-collation function</description>.
+ 
+  <dependency type="spec" value="XP40+ XQ40+"/>
+
+
+  <test-case name="fn-collation-001">
+      <description>unknown fallback</description>
+      <created by="Michael Kay" on="2024-04-03"/>
+      <test>compare('abc','aBC',collation({'fallback':'unknown'}}))</test>
```

Comment:
If the value of `fallback` needn’t be a boolean value, might `'fallback': true#0` be valid, too?